exercises/Fibonacci/Python/main.py

```diff
@@ -0,0 +1,14 @@
+n = int(input("Enter an integer :"))
+x0 = 0
+x1 = 1
+
+i=1
+while i < n/2:
+    x0=x0+x1
+    x1=x0+x1

```

Comment:
This is a challenging bit of code, it might be nice to introduce temporary variables with descriptive names like `prev_x0` and `prev_x1` and use them to avoid confusion